packages/stream_video/lib/src/call/call.dart

```diff
@@ -2137,6 +2138,62 @@ class Call {
     return const Result.success(false);
   }
 
+  Future<Result<None>> setZoom({
+    required double zoomLevel,
+  }) async {
```

Comment:
I wonder if we should name this slightly different, like `setLocalCameraZoom`. Would be really nice if at some point we can zoom into other tracks, such as the screen sharing track. Apps like Microsoft teams allow for zooming on screen shares from other participants. I do see that the `rtc.helper` in the end really uses the camera utils on the native side, so doesn't make sense to create a generic `setZoom` for every track.